packages/manager/modules/manager-pci-common/src/api/hook/usePCIFeatureAvailability.ts

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from '@tanstack/react-query';
+import { useApplicationName } from '@/hooks/useApplicationName/useApplicationName';
+import {
+  FeatureAvailabilityQueryParams,
+  getFeatureAvailability,
+} from '@/api/data/feature-availability';
+
+const useFeatureAvailability = <T extends string = string>(
+  features: T[],
+  params?: FeatureAvailabilityQueryParams,
+) =>
+  useQuery({
+    queryKey: ['feature-availability', ...features, params],
```

Comment:
i think the spread of `...features` is wrong in terms of query key and potential query cache invalidation since a join is performed to build the query on the api side

Reply:
It's a copy from pci-common, I don't know how this would cause an issue